src/fswAlgorithms/attGuidance/oneAxisSolarArrayPoint/oneAxisSolarArrayPoint.c

```diff
@@ -284,7 +284,7 @@ void oasapComputeFirstRotation(double hRefHat_B[3], double hReqHat_B[3], double
 }
```

Comment:
The message on this commit is too long and gets truncated. You can adjust and then we can merge.

Reply:
Fixed it.